packages/stream-tile/src/tile-document.ts

```diff
@@ -265,6 +265,8 @@ export class TileDocument<T = Record<string, any>> extends Stream {
         const header = headerFromMetadata(metadata)
         if (!metadata?.deterministic) {
             header.unique = uint8arrays.toString(randomBytes(12), 'base64')
+        } else if (content) {
+            throw new Error('Content specified for deterministic creation')
```

Comment:
How about `Initial content must be null when creating a deterministic document`?

Reply:
Though about mentioning that it is now a stream, but really it is a _tile document_

+1